src/tools/perf/ucp_tests.cc

```diff
@@ -295,7 +295,8 @@ class ucp_perf_test_runner {
 
 	if (m_perf.params.mem_type == UCT_MD_MEM_TYPE_HOST) {
             *((volatile uint8_t*)m_perf.recv_buffer + length - 1) = -1;
-	} else if (m_perf.params.mem_type == UCT_MD_MEM_TYPE_CUDA) {
+	} else if (m_perf.params.mem_type == UCT_MD_MEM_TYPE_CUDA
```

Comment:
Please guard each statement with () and move || to the first line

Reply:
@yosefe this doesn't apply. There's just one condition